transport-native-unix-common/src/main/c/netty_unix_filedescriptor.c

```diff
@@ -41,7 +42,7 @@ static jint _write(JNIEnv* env, jclass clazz, jint fd, void* buffer, jint pos, j
     ssize_t res;
     int err;
     do {
-       res = write(fd, buffer + pos, (size_t) (limit - pos));
+       res = send(fd, buffer + pos, (size_t) (limit - pos), 0);
        // keep on writing if it was interrupted
```

Comment:
I think it would be better to add extra method for this and not just replace the existing ones

Reply:
turning into draft and fixing this, come back soon :)